### ISSUE TITLE
Retry attribute suggestion

### ISSUE THREAD

**Issue by gavrielk** [kravietz/pam_tacplus#135]
Hi,

Due to a customer request to retry the connection attempt in case of failure or timeout expiration, we added a retry attribute and counter on the connection attempt, is it interesting to contribute to the general public?

Thanks,
Gabi

**Comment by kravietz** [comment]
@gavrielk Yes, I believe this would be definitely of interest - please submit a PR and I'll merge it.
